### ISSUE TITLE
Example for "Integrated Windows provider" has at least a typo

### ISSUE THREAD

**Issue by codewerfer** [microsoftgraph/microsoft-graph-docs#21406]
``` C#
var result = await publicClientApplication.AcquireTokenByIntegratedWindowsAuth(scopes).ExecuteAsync(); ;
// get the token and return it in your own way
return Task.FromResult(result.A);
```
Can't be correct.

But I have the strong fealing, the whole example doesn't work.

https://learn.microsoft.com/en-us/graph/sdks/choose-authentication-providers?tabs=CS#integrated-windows-provider

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 4537f77e-a8e8-e3f1-9cc4-2fd930db7ada
* Version Independent ID: 46396fa2-f7e2-322e-5e76-f9e2d17d1b48
* Content: [Choose a Microsoft Graph authentication provider - Microsoft Graph](https://learn.microsoft.com/en-us/graph/sdks/choose-authentication-providers?tabs=CS#integrated-windows-provider)
* Content Source: [concepts/sdks/choose-authentication-providers.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/concepts/sdks/choose-authentication-providers.md)
* Product: **non-product-specific**
* Technology: **microsoft-graph**
* GitHub Login: @MIchaelMainer
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by codewerfer** [comment]
Seams that somebody as already working on it: https://github.com/microsoftgraph/msgraph-snippets-dotnet/blob/main/src/SdkSnippets/Snippets/IntegratedWindowsTokenProvider.cs

**Comment by lramosvea** [comment]
@jasonjoh tagging you since I see your name here: https://github.com/microsoftgraph/msgraph-snippets-dotnet/blob/main/src/SdkSnippets/Snippets/IntegratedWindowsTokenProvider.cs

Not sure if you have an idea about this?

**Comment by codewerfer** [comment]
@lramosvea the typo is the `return Task.FromResult(result.A);`, it should be `return Task.FromResult(result.AccessToken);`

**Comment by jasonjoh** [comment]
Yes thank you for reporting this @codewerfer! I'm in the process of moving code snippets out of the Markdown source and into compilable source files to help catch typos like this. My work-in-progress is currently in the `jasonjoh/sdk-snippets` branch here: https://github.com/microsoftgraph/microsoft-graph-docs/blob/jasonjoh/sdk-snippets/concepts/sdks/choose-authentication-providers.md#integrated-windows-provider.
